Cohort1/tbd/server/server.js

```diff
@@ -10,14 +10,21 @@ const mongoose = require('mongoose');
 // useNewUrlParser: true,
 // useUnifiedTopology: true
 // fix deprecation warnings
-  mongoose.connect(process.env.MONGO_URI,{
-    useNewUrlParser: true,
-    useUnifiedTopology: true
-  })
+//////////
```

Comment:
We have the git history, so there's no need to keep commented out code. We should remove this.